README.md

```diff
@@ -89,14 +89,14 @@ $ docker run --rm -v "$PWD":/usr/src/myapp -w /usr/src/myapp \
 
 ## Contributing
 
-Please be aware this project is designed to be pushed upstream to GCC when we reach some milestones and this means we require
-contribtions to have copyright assignment in place. Please see: https://gcc.gnu.org/contribute.html
+Please be aware this project is designed to be pushed upstream to GCC when we reach some milestones, and this means we require
+contributions to have copyright assignment in place. Please see https://gcc.gnu.org/contribute.html.
 
-Not all contributions must be code, we would love to see new test cases or bugs and issues to be reported. Feel free to add any comments on open PRs
+Not all contributions must be code; we would love to see new test cases or bugs and issues to be reported. Feel free to add any comments on open PRs
 
 ## Community
 
-We can be found on all usual Rust channels such as Zulip but we also have our own channels:
+We can be found on all usual Rust channels such as Zulip, but we also have our channels:
```

Comment:
I think the original wording for this was also better. It distinguishes the "usual Rust channels" from "our own channels".

Reply:
Agreed. Reverted.